careers/styles/styles.css

```diff
@@ -211,13 +211,13 @@ button {
 .button-container > em > a.secondary,
 button {
   min-width: 133px;
-  border: 1px solid var(--static-color-black);
+  border: 1px solid var(--static-color-pink-red);
```

Comment:
is the pink-red color for buttons global across the site?

Reply:
Yes. Looks like all the buttons on caesars-careers have a pink-red colour.

Reply:
+1